src/Microsoft.ML.Core/Utilities/VBufferUtils.cs

```diff
@@ -592,9 +592,9 @@ public static void CreateMaybeSparseCopy<T>(ref VBuffer<T> src, ref VBuffer<T> d
         /// <param name="dst">Argument vector, whose elements are only read</param>
         /// <param name="res">Result vector</param>
         /// <param name="manip">Function to apply to each pair of elements</param>
-        public static void ApplyWithCopy<TSrc, TDst>(ref VBuffer<TSrc> src, ref VBuffer<TDst> dst, ref VBuffer<TDst> res, PairManipulatorCopy<TSrc, TDst> manip)
+        public static void ApplyWithCopy<TSrc, TDst>(in VBuffer<TSrc> src, ref VBuffer<TDst> dst, ref VBuffer<TDst> res, PairManipulatorCopy<TSrc, TDst> manip)
```

Comment:
>ref VBuffer<TDst> dst [](start = 75, length = 21)

/// <param name="dst">Argument vector, whose elements are only read</param>

Reply:
I will update this comment, since it is lying.

---
In reply to: [229498527](https://github.com/dotnet/machinelearning/pull/1454#discussion_r229498527) [](ancestors = 229498527)